src/github.com/couchbase/sync_gateway/rest/server_context.go

```diff
@@ -383,19 +377,59 @@ func (sc *ServerContext) InitCBGTManager() (base.CbgtContext, error) {
 
 	err = manager.Start(register)
 	if err != nil {
-		log.Printf("Manager.Start() returned error: %v", err)
+		return base.CbgtContext{}, err
+	}
+
+	if err := sc.enableCBGTAutofailover(
+		cbgt.VERSION,
+		manager,
+		cfgCb,
+		uuid,
+		couchbaseUrl,
+		db.KSyncKeyPrefix,
+	); err != nil {
 		return base.CbgtContext{}, err
 	}
 
 	cbgtContext := base.CbgtContext{
 		Manager: manager,
-		Cfg:     cfg,
+		Cfg:     cfgCb,
 	}
 
 	return cbgtContext, nil
 
 }
 
+func (sc *ServerContext) enableCBGTAutofailover(version string, mgr *cbgt.Manager, cfg cbgt.Cfg, uuid, couchbaseUrl, keyPrefix string) error {
+
+	cbHearbeater, err := autofailover.NewCouchbaseHeartbeater(
```

Comment:
Typo in the variable name.  Not critical to fix obviously, but in case you end up making other changes.